tests/general/InvalidLinkBearTest.py

```diff
@@ -122,3 +127,35 @@ def test_check_prerequisites(self):
         with requests_mock.Mocker() as m:
             m.add_matcher(custom_matcher)
             self.assertTrue(InvalidLinkBear.check_prerequisites())
+
+    def test_redirect_threshold(self):
+
+        long_url_redirect = """
+        https://bitbucket.org/api/301
+        https://bitbucket.org/api/302
+        """.splitlines()
+
+        short_url_redirect = """
+        http://httpbin.org/status/301
+        """.splitlines()
+
+        self.assertResult(valid_file=long_url_redirect,
+                          invalid_file=short_url_redirect)
+
+    def test_ignore_regex(self):
```

Comment:
this test doesn't belong into ths commit does it? (https://github.com/coala-analyzer/coala-bears/pull/458/commits/a3bd69b696691138ab87b87c8634dadf219fe193)